neverbleed.c

```diff
@@ -837,7 +888,11 @@ static EVP_PKEY *create_pkey(neverbleed_t *nb, size_t key_index, const char *ebu
     exdata->key_index = key_index;
 
     rsa = RSA_new_method(nb->engine);
-    RSA_set_ex_data(rsa, 0, exdata);
+    NEVERBLEED_MULTITHREAD_ONCE({
+        ecdsa_exdata_idx = RSA_get_ex_new_index(0, NULL, NULL, NULL, ecdsa_exdata_free_callback);
```

Comment:
I believe you meant `rsa_exdata_idx` here (and `ecdsa_exdata_idx` in `ecdsa_get_privsep_data`)?